paper/paper.md

```diff
@@ -31,54 +31,54 @@ bibliography: paper.bib
 # Summary
 
 Dynamical systems such as those derived from models of fluid flows show transition behavior associated with fluid flow instabilities [@wubs:23].
-Examples are dynamical systems from ocean models which show transition behavior due to slow changes in parameters representing the surface forcing [@westen:24].
+Examples are dynamical systems from ocean models in which transition behavior is caused by slow changes in parameters representing the surface forcing [@westen:24].
 Such changes in the stability of the fluid flow can be studied by performing numerical simulations with a model and observing its transient behavior after a certain time.
 This is, however, computationally very expensive, and in many cases infeasible.
-Instead, so called continuation methods exist, that are able to trace stable and unstable steady states in parameter space, without having to perform the expensive transient simulations [@dijkstra:05].
+Instead, so called continuation methods are able to trace stable and unstable steady states in parameter space, obviating expensive transient simulations [@dijkstra:05].
 The `TransiFlow` Python package implements a continuation framework in which fluid flow problems can be studied with the help of several computational back-ends that can, based on the needs of the user, easily be switched between.
```

Comment:
I'd add here a paragraph on what the model can actually do (like the first paragraph in the docs) and what its
constraints are. To me, it's not really clear which "fluid flow problems" can be
studied (only later "canonical" problems are mentioned), and what said "needs of the user" can be, which the code caters for. I.e., I'd appreciate an answer to the possible question: "can I use this code for my project?"
See also related comment below